### PR TITLE
[js-transform] inline java script support

### PR DESCRIPTION
**Pull request by paulianttila**
Signed-off-by: Pauli Anttila <pauli.anttila@gmail.com>

### CODE REVIEW

**Comment by paulianttila** [comment]
Recently written lot of java script transformation rules and file based broach is not very handy for simple script where input is just e.g. dived by 100. This introduce inline possibility where script can be directly given without writing dedicated java script file. 

Some earlier discussion https://github.com/eclipse-archived/smarthome/issues/4318

**Comment by paulianttila** [comment]
@openhab/add-ons-maintainers as this is rather simple PR and no comments so far, so is there some kind of hesitation to accept this PR?

**Comment by fwolter** [comment]
I've never been in touch with JavaScript and OH. That's why I didn't approve it. Maybe @wborn can take a look.

File: bundles/org.openhab.transform.javascript/src/main/resources/OH-INF/config/javascriptProfile.xml [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 	<config-description uri="profile:transform:JS">
 		<parameter name="function" type="text" required="true">
-			<label>JavaScript Filename</label>
-			<description>Filename of the JavaScript in the transform folder. The state will be available in the variable
-				\"input\".</description>
+			<label>JavaScript Filename or inline script</label>
```
Comment (kaikreuzer) [inline bundles/org.openhab.transform.javascript/src/main/resources/OH-INF/config/javascriptProfile.xml:9]:
```suggestion
			<label>JavaScript Filename or Inline Script</label>
```

**Review by kaikreuzer — APPROVED** [review]
Lgtm and it is imho indeed a nice feature for simple use cases. Thanks!

**Pull request merged**